UI/src/components/templates/capone.html

```diff
@@ -9,7 +9,7 @@
 
                 <div class="pipeline-toggle btn-group"
 
-                     ng-show="template.hasComponents(ctrl.dashboard, ['Deployment'])">
+                     ng-show="template.hasComponents(ctrl.dashboard, ['SCM'])">
```

Comment:
You mean this should be visible if just somebody creates a Dashboard, BTW this is how it will look

![image](https://cloud.githubusercontent.com/assets/6624821/18725984/71468dba-8010-11e6-9c29-07584d5d41cf.png)